metrics/grafana/pd.json

```diff
@@ -7909,7 +7909,7 @@
           "tableColumn": "",
           "targets": [
             {
-              "expr": "pd_checker_patrol_regions_time{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"} != 0",
+              "expr": "max(pd_checker_patrol_regions_time{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"})",
```

Comment:
I don't think max is right

Reply:
![image](https://github.com/tikv/pd/assets/19542290/e48aac71-e510-4f01-ae97-af36eacb6d1e)
How about showing all results? Or do you have any good ideas?

Reply:
will it auto-hidden if no scheduling service?

Reply:
It may be complex to return the latest values from multiple time series. https://stackoverflow.com/questions/76930324/return-most-recent-value-of-multiple-time-series-in-prometheus-promql

Reply:
If we're sure that patrol won't be reported by different nodes, perhaps we can use sum() directly.